<launch>
   <node pkg="cv_camera" type="cv_camera_node" name="cv_camera">
      <param name="device_id" value="1" />
      <param name="rate" value="10" />
   </node>
	
	<node name="object_checker" pkg="open_manipulator_visual_servoing" type="object_checker" output="screen"/>
</launch>
